### PR TITLE
Fix/collateral loan panels data

### PR DESCRIPTION
**Pull request by tuan-tl**
- Fix deposited amount to be calculated based on sICX ratio
- Fix borrowed amount to get from bnUSD asset
- Fix bug on load for empty account that caused app crash

### CODE REVIEW

File: src/app/components/home/LoanPanel.tsx [hunk 1]
```diff
@@ -25,6 +26,7 @@ const LoanPanel = () => {
 
   const stakedICXAmount = useDepositedValue();
   const loanBorrowedValue = useLoanBorrowedValue();
+  const walletBalance = useWalletBalanceValue();
   const totalLoanAmount = stakedICXAmount.div(4).minus(loanBorrowedValue);
```
Comment (WeiLi512) [inline src/app/components/home/LoanPanel.tsx:30]: totalLoanAmount = (stakedICXAmount / 4) - loanBorrowedValue
I think it is not correct formula.
totalLoanAmount = (stakedICXAmount /5)

File: src/app/components/home/LoanPanel.tsx [hunk 2]
```diff
@@ -254,21 +183,24 @@ const LoanPanel = () => {
           </Typography>
 
           <Typography variant="p" fontWeight="bold" textAlign="center" fontSize={20}>
-            0 bnUSD
+            {formattedAmounts[Field.LEFT]} bnUSD
```
Comment (WeiLi512) [inline src/app/components/home/LoanPanel.tsx:186]: I think it is not correct.
Please reference how the prototype works

File: src/app/components/home/LoanPanel.tsx (hunk 1 above)
Comment (gedanziger) [reply src/app/components/home/LoanPanel.tsx:30]: max you can borrow is 1/4 of collateral.
Max you can borrow and receive reward is 1/5 of collateral, so at 1/4 you can still borrow but not get a reward.

To get remaining amount you can borrow, take max you can borrow, and subtract already borrowed amount

Comment (WeiLi512) [reply src/app/components/home/LoanPanel.tsx:30]: I thought the collateral ratio was 500%.

Comment (WeiLi512) [reply src/app/components/home/LoanPanel.tsx:30]: I think we need to calculate the stakedICXAmount in USD.
stakedICXAmount * (sICX/USD price) / 4 - loanBorrowedValue

Comment (WeiLi512) [reply src/app/components/home/LoanPanel.tsx:30]: If the sICX(or ICX) price drops, the borrowed amount is bigger than the total borrowable available amount. In this case, the available amount is minus. In this case, how will handle that UI?
image

Comment (gedanziger) [reply src/app/components/home/LoanPanel.tsx:30]: if less than 0, you can display 0

**Pull request merged**